include/openthread/netdata.h

```diff
@@ -274,6 +274,31 @@ uint8_t otNetDataGetVersion(otInstance *aInstance);
  */
 uint8_t otNetDataGetStableVersion(otInstance *aInstance);
 
+/**
+ * Check if the steering data allows a Joiner.
+ *
+ * @param[in]  aEui64             A pointer to the Joiner's IEEE EUI-64 or NULL for any Joiner.
+ *
+ * @retval OT_ERROR_NONE          The joiner is in the bloom filter.
```

Comment:
```suggestion
 * @retval OT_ERROR_NONE          The Joiner is allowed by the steering data.
```

bloom filter is actually implementation detail which should not be exposed?